src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Formatting/FormattingHandler.ts

```diff
@@ -46,8 +46,24 @@ export class FormattingHandler {
                 virtualHtmlUri,
                 formattingParams.options);
 
+            if (textEdits === undefined) {
```

Comment:
This check should (hopefully) resolve https://github.com/dotnet/razor/issues/7038